content/products/dolthub/api/database.md

```diff
@@ -298,3 +298,20 @@ headers = {
 {% swagger src="../../../.gitbook/assets/dolthub-api/listreleases.json" path="/{owner}/{database}/releases" method="get" %}
 [listreleases.json](../../../.gitbook/assets/dolthub-api/listreleases.json)
 {% endswagger %}
+
+
+## List operations
+
```

Comment:
I think you need a description here of what this is and why you'd use it before you go into the example.